client_release.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Requires Jenkins Go Plugin to be enabled in build environment
+
+#exit on error
+set -e
+set -x
+
+export GOPATH=$(pwd)
+export GITHUB_USER=WycliffeAssociates
+export GITHUB_TOKEN=$1
+export GITHUB_REPO=te-release
+export PATH=$PATH:$GOPATH/bin
+
+#Get the latest tag
+TAG=$(git describe --tags --abbrev=0)
+
+#Make sure we actually grabbed a tag
+if [ -z $TAG ]; then exit
+fi
+
+#Check if there is a release for the latest tag
+#Exit if a release already exists
+DOES_RELEASE_EXIST=$(curl -s https://api.github.com/repos/wycliffeassociates/te-release/releases/tags/$TAG | grep '"message": "Not Found"')
+if [ -z "$DOES_RELEASE_EXIST" ]; then exit;
+fi
+
+mkdir clients
+cp translationExchangeAndroid/app/build/outputs/apk/release/app-release.apk clients/te_android_client.apk
+cp translationExchange/dist/*.exe clients/te_windows_client.exe
```

Comment:
I tried that, but it just names it `*.exe` instead of preserving the name. `cp translationExchange/dist/*.exe clients/` preserves the name though, so I'll use that.

Reply:
done